### ISSUE TITLE
Feature request: Vote to disable dummies

### ISSUE THREAD

**Issue by bencie** [ddnet/ddnet#6884]
A lot of people don't like playing with dummy players in team 0, so what if players could vote to disable connecting new dummies to the server? It'd be up to the whole server to make the vote without veto, and could bring the community together better, because you know you'll only play with other players, and no ego dummy players.

**Comment by heinrich5991** [comment]
I don't think this is something the server should concern itself with to be honest.

**Comment by l-ouis** [comment]
> I don't think this is something the server should concern itself with to be honest.

could you elaborate?
I think it's a good idea because it caters to both people who enjoy dummies and those who don't

**Comment by heinrich5991** [comment]
> and no ego dummy players.

Can you elaborate on the problem of playing with people who are also bringing a dummy along? I think I've generally had good experiences with people bringing their dummy along, usually insanely skilled players who would stay behind and save the entire team if everyone else failed.

> I don't think this is something the server should concern itself with to be honest.

I'm not sure how to effectively enforce this to be honest. Perhaps just disabling the dummy button would work. If not, though, and people start using other clients or just two clients at once, then how would we deal with that? The server could no longer help with that problem.

**Comment by heinrich5991** [comment]
> Well, there's already a server command to limit how many players can connect on one ip, not sure how this setting works if there are already dummies on the server. If it doesn't kick those dummies (so that dummies playing mid run don't get kicked), then the vote could just toggle this setting (sv_max_clients_per_ip).

This doesn't work because it prevents people in the same household from playing on the same server. That'd be a very bad thing IMO.

**Comment by refeax** [comment]
> You could check for dummy binds (hammerfly) and kick anyone flagged with the aim snap

if you're talking about an autodetect including the antibot system, it wouldn't be very good because the antibot is currently too inconsistent to correctly tell apart such things

**Comment by heinrich5991** [comment]
We have a server flag servers can use to make the client disable dummies. It's best-effort only. I don't think we should enable it on official DDNet servers, also because it'll make people look for workarounds.

**Comment by l-ouis** [comment]
Hi, got reminded of this topic. There's a lot of complaining and requests for servers with such a flag (especially in the discord rn). 

I don't think there's any harm in using this setting on a portion of DDNet servers and seeing if people actually play on them. We'd only have to clearly label them as non-dummy so players don't get confused.

**Comment by KebsCS** [comment]
So turns out ddnet already has a server that limits to 1 client per IP? `DDNet GER - MULTEASYMAP BLOCKER #1
Address: ddnet://45.141.57.22:8301`
 Why was this solution used if "server can already disable dummies" https://github.com/ddnet/ddnet/pull/11260#issuecomment-3537516599 and "it prevents people in the same household from playing on the same server. That'd be a very bad thing IMO." https://github.com/ddnet/ddnet/issues/6884#issuecomment-1640256046

**Comment by trikiman** [comment]
about  4-5 years ago i ask smth like this.
i created map with is really good idea but dummy destroy it. 
when i ask about disabling dummy on my map every body said "never".
So look like i can continue my old map.

**Comment by heinrich5991** [comment]
> So look like i can continue my old map.

This issue has not been accepted.

**Comment by trikiman** [comment]
> > So look like i can continue my old map.
> 
> This issue has not been accepted.

why this sissue still open then?
